docs/_includes/DeveloperGuide/Requirements/02_use_cases.md

```diff
@@ -61,7 +61,7 @@ Actor: New User
 
 **Extensions**
 - **2a.** The system finds no match.
-  - 2a1. The system displays a message indicating no results.
```

Comment:
The system does still display some form of message such as `0 persons listed!` as feedback. May want to edit accordingly